environment.yml

```diff
@@ -0,0 +1,25 @@
+name: pyvista-env
+channels:
+  - conda-forge
+dependencies:
+  - python=3.7
```

Comment:
vtk 8.0 is only available for python 3.5 and 3.6. Maybe put here the same requirement as in the readme (>=3.5) and then let conda figure out a combination that works?